src/borrow_check/drop_check.md

```diff
@@ -146,7 +146,7 @@ a type is contained in a larger local. **This is inconsistent, and
 should be fixed: an example [for
 arrays](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=8b5f5f005a03971b22edb1c20c5e6cbe)
 and [for
-`PhantomData`](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=44c6e2b1fae826329fd54c347603b6c8).**[^core]
```

Comment:
Found what was supposed to be core.

`[^core]: This is the core assumption of [#110288] and [RFC 3417].`

Can you add that?

Reply:
Sure, it should be updated now.